### DISCUSSION TITLE
Pterodactyl panel

### DISCUSSION THREAD

**Discussion by Furanzou** [vilhelmprytz/pterodactyl-installer#281] [Q&A]
So I’ve recently using goormide right now and wondering on people why are they using it for pterodactyl panel?, I mean if there gonna run a game server don’t they need to install docker?

**Comment by Linux123123** [comment]
This is more of a discussion topic then an issue.

**Comment by Furanzou** [comment]
Yeah I know and I’ve been meaning to know on why people suggest using it if resources aren’t met for example “docker”

**Comment by Furanzou** [comment]
Well I setted the pterodactyl but it’s giving me a blank page
